### PR TITLE
Refactor structure

### PR DESCRIPTION
**Pull request by pgjones**
This is a fairly large refactor to account for the change in flake8-import-order from a single style to multi style checker. This pull request will remain open for testing and discussion.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,11 @@
+0.9 (Unreleased)
+----------------
+
+* Drop pep8 requirement and replace with pycodestyle.
+* Support Flake8 3.0 (alongside Flake8 2.X).
+* Drop Python2.6 compatibility.
```
Comment (sigmavirus24) [inline CHANGELOG.rst]: :+1: :100:

Comment (public) [inline CHANGELOG.rst]: +1

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+* Refactor code structure, fixing bugs in the process.
```
Comment (public) [inline CHANGELOG.rst]: Any idea which bugs? :-)

Comment (pgjones) [inline CHANGELOG.rst]: Good point, I've updated the changelog. It is #76 and #49, #50 I think are fixed.

**Comment by pgjones** [comment]
I've tested on:
 - [x] cryptography (cryptography style)
 - [x] flake8 (google style)
 - [x] Propriety Smarkets code (smarkets style)
 - [x] mypy (pep8) https://github.com/python/mypy/pull/1883
I'm therefore happy to merge.

**Pull request merged**